src/views/report/ScriptReportDetail/ScriptExecutionDetailActions.tsx

```diff
@@ -296,7 +296,7 @@ function ScriptExecutionDetailActions<ColumnNames>({
                                                     {parameter.rawValue}
                                                     {
                                                         (parameter.name === 'request') ? (
-                                                            <Box display="flex">
+                                                            <Box>
```

Comment:
But now, there is no difference with the previous commit. Did you check on the UI if the button in all the way on the right and the name of the metadata on the left ? Please re-read what I've said on your previous commit, something is misisng.